### PR TITLE
Update go to plan compare text and cucumber.

### PR DESCRIPTION
**Pull request by ZASMan**
https://www.pivotaltracker.com/n/projects/2502930/stories/180140814

### CODE REVIEW

File: features/step_definitions/integration_steps.rb [hunk 1]
```diff
@@ -873,6 +873,14 @@ def non_dc_office_location
   end
 end
 
+And(/^.+ clicks Go To Plans link$/) do
```
Comment (markgoho) [inline features/step_definitions/integration_steps.rb:876]: are we hard-coding english text into a test?

Comment (ZASMan) [reply features/step_definitions/integration_steps.rb:876]: Ok good point I found the file where its rendered and used the translation as a link clicking reference and it passes

Comment (ZASMan) [reply features/step_definitions/integration_steps.rb:876]: Fixed

**Review by vkghub — APPROVED** [review]
Text changes as per the request.

**Pull request merged**